docs/essentials/collections/sortedCollectionTypes.md

```diff
@@ -1 +1,42 @@
+   
+*   Elements are sorted according to a [System.Collections.IComparer](http://dotnet.github.io/api/System.Collections.IComparer.html) implementation (for nongeneric **SortedList**) or a [System.Collections.Generic.IComparer&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.IComparer%601.html) implementation (for the two generic classes).  
+   
+ *   Each class provides properties that return collections containing only the keys or only the values.  
+   
+The following table lists some of the differences between the two sorted list classes and the [SortedDictionary&lt;TKey, TValue&gt;](http://dotnet.github.io/api/System.Collections.Generic.SortedDictionary%602.html) class.  
+   
+ **SortedList** nongeneric class and **SortedList&lt;TKey, TValue&gt;** generic class | **SortedDictionary&lt;TKey, TValue&gt;** generic class  
+ --------------------------------------------------------------------------------- | ------------------------------  
+ The properties that return keys and values are indexed, allowing efficient indexed retrieval. | No indexed retrieval.  
+ Retrieval is O(log n). | Retrieval is O(log n).  
+ nsertion and removal are generally O(n); however, insertion is O(1) for data that are already in sort order, so that each element is added to the end of the list. (This assumes that a resize is not required.) | Insertion and removal are O(log n).  
+ Uses less memory than a **SortedDictionary&lt;TKey, TValue&gt;**. | Uses more memory than the **SortedList** nongeneric class and the **SortedList&lt;TKey, TValue&gt;** generic class.  
+  
+ For sorted lists or dictionaries that must be accessible concurrently from multiple threads, you can add sorting logic to a class that derives from [ConcurrentDictionary&lt;TKey, TValue&gt;](http://dotnet.github.io/api/System.Collections.Concurrent.ConcurrentDictionary%602.html).  
+  
+ > **Note**  
+ >   
+ >For values that contain their own keys (for example, employee records that contain an employee ID number), you can create a keyed collection that has some characteristics of a list and some characteristics of a dictionary by deriving from the [KeyedCollection&lt;TKey, TItem&gt;]() generic class.  
+   
+ The [SortedSet&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.SortedSet%601.html) class provides a self-balancing tree that maintains data in sorted order after insertions, deletions, and searches. This class and the [HashSet&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.HashSet%601.html) class implement the [ISet&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.ISet%601.html) interface.  
+   
+ ## See Also  
```

Comment:
GitHub does not format this as a heading, I believe it's because of the leading space.